pallets/runtime/tests/src/settlement_test.rs

```diff
@@ -1984,6 +2008,16 @@ fn basic_fuzzing() {
             }
 
             if fail {
+                // Ensure tokens stayed locked when instruction failed
+                for user in &users {
+                    assert_eq!(
+                        *locked_assets.get(&user.did).unwrap(),
+                        <PortfolioLockedAssets<TestStorage>>::iter_prefix(
+                            PortfolioId::default_portfolio(user.did,)
+                        )
+                        .collect::<Vec<_>>()
+                    );
+                }
```

Comment:
The `locked_assets` snapshot was saved right before the `next_block()` call with no other actions?  I wouldn't expect the locked assets to have changed.